### ISSUE TITLE
Duplicate symbol for arm64

### ISSUE THREAD

**Issue by janaraj** [box/box-ios-preview-sdk#6]
Thanks much for the sdk, I was trying to integrate and getting the below error. 

Even with sample project I'm facing the same error, probably I'm missing something, could you please help ?

duplicate symbol l005 in:
    /Users/jana/Workspace/Projects/Box/box-ios-preview-sdk/BoxPreviewSDKSampleApp/Pods/box-ios-preview-sdk/BoxPreviewSDK.framework/BoxPreviewSDK(PSPDFCheckmarkOverlayView.o)
    /Users/jana/Workspace/Projects/Box/box-ios-preview-sdk/BoxPreviewSDKSampleApp/Pods/box-ios-preview-sdk/BoxPreviewSDK.framework/BoxPreviewSDK(gost_pmeth.o)
duplicate symbol l002 in:
    /Users/jana/Workspace/Projects/Box/box-ios-preview-sdk/BoxPreviewSDKSampleApp/Pods/box-ios-preview-sdk/BoxPreviewSDK.framework/BoxPreviewSDK(PSPDFCommonMetrics.o)
    /Users/jana/Workspace/Projects/Box/box-ios-preview-sdk/BoxPreviewSDKSampleApp/Pods/box-ios-preview-sdk/BoxPreviewSDK.framework/BoxPreviewSDK(gost_sign.o)
ld: 775 duplicate symbols for architecture arm64

**Comment by boxtcarpel** [comment]
Hi @janaraj Thank you for using the SDK and for reaching out!
I am unable to reproduce this issue.
I checked out a clean clone of the SDK, updated the pod in the sample app project, and ran the sample app. It built and ran without issue.

Are you using the latest CocoaPods version?

**Comment by janaraj** [comment]
Thanks for checking, yes I'm using the latest CocoaPods, 
pod --version
0.39.0

I tried many times , always ended up with the same issue. Let me do some more research and update you, meanwhile if you could think of something please do let me know.

**Comment by janaraj** [comment]
Just in case, steps that I tried, 
1.) downloaded the zip file,
2.) executed pod install under sample project
3.) opened xcworkspace and did a build

Then it failed with duplicate error.

Also tried by doing github clone as well, still same issue

**Comment by boxtcarpel** [comment]
@janaraj thanks for following up and trying so diligently to get this working.

To be clear, we do have a known issue right now (that we are in the process of fixing) where the header search path is not getting set correctly. So for me, when doing a clean build of the current sample app as-is it doesn't build for that reason. Once I fix that (temporarily by changing the preview-sdk entry of the header search paths in the sample app target to 'recursive' from 'non-recursive'), the sample app builds and runs with no duplicate symbols errors.
Are you seeing this at all? If not, then it would seem our environments are still off from each other.

**Comment by boxtcarpel** [comment]
By the way, I am also on the same version of CocoaPods (0.39.0), so it must be something else that is different between our environments.

**Comment by janaraj** [comment]
Thanks much for following up, I finally able to figure out the issue, 

the import to BoxPreviewSDK was throwing error, so as suggested by another member I used absolute path. The problem was because of this, the BoxPreviewSDK.h inside the Pods folder was also throwing this error, so I had to include the absolute path inside this as well, but instead of changing inside the versions folder, I made the change in BoxPreviewSDK.framework folder, this had triggered the whole issue.

I reverted back to old import statement and made the changes inside the versions file. Now I'm able to run the demo app.

Hope the import statement issue gets fixed in the coming version. For now I'm able to run the app. 

Thanks much for your help.
